migrations/20200608133617-update_list_observation_solutions_query.js

```diff
@@ -0,0 +1,35 @@
+const Uuid = require('cassandra-driver').types.Uuid;
+
+module.exports = {
+  async up(db) {
+    global.migrationMsg = "Update list observation solutions query";
+    
+    
+    if (!cassandra) {
+      throw new Error("Cassandra connection not available.");
+    }
+
+    const query = 'SELECT id FROM reports.druidqueries WHERE qid = ? ALLOW FILTERING';
+    const result = await cassandra.execute(query, [ 'list_observation_solutions_query' ], { prepare: true });
```

Comment:
@deepakr123 - Let's have 2 main queries, programs_list_query and solution_list_query which is common for observation, institutional and individual assessment

Reply:
resolved